guide/payments/introduction.md

```diff
@@ -0,0 +1,86 @@
+
+## Bitcoin is Global
+
+We are for designing the next evolution of money. One which is internet based, and global.
+
+> **Getting Started / Why bitcoin is unique**
+>
+> Bitcoin can be used by anyone with an internet connection, which is available to around [53.6% of the world population](https://en.wikipedia.org/wiki/Global_Internet_usage), a number that is growing quickly. Some aspects of bitcoin, like key and address generation, even work without being online. This makes it the first truly globally available form of money that is unbound by country borders. People from around the world can freely transact without having to rely on middlemen like currency exchanges and local payment services.
+
+## Bitcoin is extensible, it's not just a payment network
+
+It should be noted, that bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrasutructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
+
+## Bitcoin is secure and accessible to all
+
+Some people may have the perception that bitcoin is anonymous. This isn't quite accurate because all transactions are public so anyone can see the entire history of transfers that ever happened going back to the very first one on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+
+#### Why are transactions public?
+
+They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of bitcoin.
+
+#### What about privacy?
+
+Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still super important though since every time you make a payment the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
+
+We should inform our users of these risks but also provide mechanisms that help them gain an acceptable amount of privacy[^1].
+
+<<<<<<< HEAD
+
+## Bitcoin is not too slow for payments
+
+Transactions made on-chain can take minutes or days to confirm — depending on the fee you pay. This is because they get secured by the entire network of nodes and miners which exchange transactions they receive from wallets and other services with each other. Every 10 minutes miners validate the payments within the transactions and publish a block. For this service, they typically take the transactions with the highest fees.
```

Comment:
This seems a bit techy. What's on-chain? Did reader had a resource before this to intro themselves with the concept, if so then it's fine.

Reply:
Might be also good to put this in perspective with the traditional banking system. From the [Wells Fargo FAQ](https://www.wellsfargo.com/help/online-banking/transfers-faqs/):

> Payments to your Wells Fargo credit account from a non-Wells Fargo account usually take 3-4 business days to complete.

So yes, paying with credit card at the grocery store is a bit impractical. But for many other uses, it's pretty hella fast for a global payment network.

Reply:
@pavlenex do you think the intro of this page would be a ok place to introduce it?

@GBKS I kind of have something similar below:

> This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone across the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.

I learned that the local bank transfer speed / cost vary quite a bit globally. For example within Indonesia and Nigeria payments between any bank is instant, but in other places it could be 2-5 days as you mentioned. The one that always takes long is with the SWIFT network though. See next line

> This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone accross the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.

Reply:
> @pavlenex do you think the intro of this page would be a ok place to introduce it?

Good question. Personally, I'd keep it separate as user dives deep into the guide. I am scared that we may overwhelm people with the concept in the introduction. , I'd explain the LN once basics of the on-chain are layed out. I kinda like the [structure Daniel suggested](https://github.com/BitcoinDesign/Guide/discussions/98#discussioncomment-284887). What is your view on this, @johnsBeharry?

Reply:
During conversations developing/testing payment content in the mentorships calls I've had, participants would quite early unknowingly raise questions relating to lightning (speed and cost etc). This is the point where some people even look at other networks because "bitcoin is too slow" (hence why I put that heading). I tried to capture the flow of those sessions on this page. It doesn't mean I will immediately be writing up lightning content though that research is still on-going, but giving awareness of it at this point makes sense to me so the reader has a more fuller understanding of the ecosystem.

During the design process people first Googling screenshots of payment and bitcoin apps. I even spoke with a designer who designed a bitcoin wallet but never used one themselves.

Admittedly, this page has grown quite long, and the reading experience certainly needs to be improved (i hear you on more visuals for sure). Perhaps it could also be moved entirely to `guide/payments/networks` or `guide/payments/getting-started` I'm happy if we also move more generally applicable parts over to `guide/getting-started` then payments could focus more on the user flow. Thoughts? @GBKS @pavlenex